src/pages/City/City/City.tsx

```diff
@@ -278,17 +279,31 @@ const City = () => {
     setAdminsAll(response.data.administration)
   }
 
-  function seeSkipModal(followerID: number) {
-    return Modal.confirm({
-      title: "Ви впевнені, що хочете покинути дану станицю?",
-      icon: <ExclamationCircleOutlined />,
-      okText: 'Так, покинути',
-      okType: 'primary',
-      cancelText: 'Скасувати',
-      maskClosable: true,
-      onOk() { removeMember(followerID) }
+  async function seeSkipModal(followerID: number) {
```

Comment:
From my point of view, `seeSkipModal` is bad name, would be better to name it as: `showSkipModal`

Also I believe you could move that modal in some common component and just reuse it in City and Club components